bin/runner.js

```diff
@@ -19,14 +19,17 @@ try {
 
         throw err;
       })
+      .then(() => runner.launchMobileEmulator())
```

Comment:
this shouldn't be done here. it should follow the same path as the selenium server